tests/unit/test_extensionPoints.py

```diff
@@ -128,7 +129,7 @@ def handlerMethod(self, a, **kwargs):
 
 	def test_instanceMethodHandlerTakesParams_givenPositional(self):
 		"""Test to ensure that a instance method handler gets the correct arguments, including implicit "self"
-		Handler takes a parameter.
+		Handler takes a positional parameter.

```

Comment:
I think this confuses things, I agree that the terminology seems to be quite inconsistent. I tend to think about the parameters from two different perspectives. The function definition perspective, and the caller perspective.
- From the function definition, in this case `handlerMethod` has one parameter, it has no default value, the function does not take a variable number of positional arguments, or a varliable number of keyword arguments. 
- From the caller perspective, `handlerMethod` is called with a single positional argument in this case. It would also be acceptable to call it with a single keyword argument `a="a value"`

I think there is often confusion on the function definition side, resulting in a parameter with a default value being thought of as a keyword argument, when it can actually be called using a positional argument, a keyword argument, or be missing entirely.